__tests__/Settings.test.js

```diff
@@ -40,6 +40,10 @@ describe("Form Settings Page", () => {
     expect(screen.queryByTestId("confirmDelete")).toBeInTheDocument();
 
     await user.click(screen.queryByTestId("confirmDelete"));
+    expect(mockedAxios.mock.calls.length).toBe(1);
+    expect(mockedAxios).toHaveBeenCalledWith(
+      expect.objectContaining({ url: "/api/templates", method: "DELETE" })
+    );
     waitFor(() => {
```

Comment:
waitFor needs an await here. Since you're already making changes to this code would appreciate if you add this as well